pkg/policy/selectorcache.go

```diff
@@ -258,6 +267,27 @@ func (s *selectorManager) IsWildcard() bool {
 	return s.key == wildcardSelectorKey
 }
 
+// must be holding selectorcache mutex
+func (s *fqdnSelector) NotifyAdded() {
+	// Make the user (FQDN subsystem) aware of this selector.
+	if s.user == nil {
```

Comment:
Understanding that the DNS poller is maybe going to get removed soon, maybe rename the `user` here as `dnsproxy`? That way the code would be more self explanatory.

Reply:
Sure, happy to do that.